VISION.md

```diff
@@ -116,7 +116,7 @@ Artichoke will include implementation-agnostic [C APIs][a-c-api] targeting:
 [file system backends]:
   https://github.com/artichoke/artichoke/labels/A-filesystem
 [virtual file system]:
-  https://artichoke.github.io/artichoke/artichoke_backend/fs/index.html
+  https://artichoke.github.io/artichoke/artichoke_backend/load_path/index.html
```

Comment:
this breakage was introduced here:

- https://github.com/artichoke/artichoke/pull/1140